datalad/core/distributed/clone.py

```diff
@@ -582,20 +582,136 @@ def postclonecfg_ria(ds, props):
     # and was enabled (there could be RIA stores that actually only have repos)
     # make this function be a generator
     ria_remotes = [s for s in ds.siblings('query', result_renderer='disabled')
-                   if s.get('annex-externaltype', None) == 'ora'
-    ]
+                   if s.get('annex-externaltype', None) == 'ora']
     if not ria_remotes:
```

Comment:
I know that this isn't introduced here, but shouldn't the entire lookup operation be conditions upon the actual presence of _any_ ora special remote configuration, i.e. look into `AnnexRepo.get_special_remote()`. That would be cheap, but all the remote talking is relatively expensive. For example, the HCP datasets live in a RIA store but none has an ORA remote configured. It already takes 5+ hours to clone them. This would make it even longer.

Reply:
Agree. But to be solved separately, right?

Reply:
I can fix that in master right now. Conflicts will then have to be resolved in here. I am OK with either approach.

Reply:
No, I take that back. The problem doesn't exist in master. It is introduced here.

Reply:
I pushed a fix in here.

Reply:
I thought replacing the `siblings` call right away and collect info on all special remotes (enabled or not) in one call. Anyways - your commit solves the bigger issue. Agree.